lib/matplotlib/dates.py

```diff
@@ -568,7 +568,8 @@ def drange(dstart, dend, delta):
 
 class DateFormatter(ticker.Formatter):
     """
-    Format a tick (in seconds since the epoch) with a `strftime` format string.
+    Format a tick (in seconds since the epoch) with a
+    `~datetime.datetime.strftime` format string.
```

Comment:
This isn't even correct is it?  Its decimal days since the epoch.

Reply:
indeed...